node/src/components/linear_chain_sync/operations.rs

```diff
@@ -123,35 +123,39 @@ where
     Ok(deploy)
 }
 
-/// Get trusted block headers; falls back to genesis if none are available
-async fn get_trusted_validator_weights<REv, I>(
+/// Returns the genesis validator weights, by public key.
+fn get_genesis_validators(chainspec: &Chainspec) -> BTreeMap<PublicKey, U512> {
+    chainspec
```

Comment:
This is basically a duplicate of some code in the [`NetworkConfig` inherent impl][1].

I inlined it before, but we should probably make method on `AccountsConfig` and dedup the other places we do this.

[1]: https://github.com/xcthulhu/casper-node/blob/76f8f6cd9a254527a74bc0a5c239c708a2af577c/node/src/types/chainspec/network_config.rs#L30-L30

Reply:
That change doesn't seem to be on `feat-fast-sync` yet. Are you going to merge it soon? Otherwise I'll probably just copy that one method?

Reply:
As discussed, I copied the method from `dev`.